FAQ/faq01.md

```diff
@@ -0,0 +1,2 @@
+# Why Data Provenance Standards?
+Data provenance standards establish criteria for documenting the origin and lifecycle of data, which includes details on how data is collected and how it may be used. This transparency is crucial for driving accuracy, reliability and trustworthiness of data, across all industries.
```

Comment:
Please distribute sentences across lines and separate structural elements with space accordingly. This way we use markdown as intended and have less noisy diffs.